### PR TITLE
ACS-3214 Update rules.

### PR DESCRIPTION
**Pull request by tpage-alfresco**

### CODE REVIEW

File: packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/UpdateRulesTests.java
Comment (tpage-alfresco) [inline packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/UpdateRulesTests.java:136]: Don't merge before #1251 has been merged and this line has been removed.

File: remote-api/src/main/java/org/alfresco/rest/api/nodes/NodeRulesRelation.java
Comment (krdabrowski) [inline remote-api/src/main/java/org/alfresco/rest/api/nodes/NodeRulesRelation.java:140]: Here probably should be "update" instead of "create".

**Review by krdabrowski — APPROVED** [review]

**Comment by krdabrowski** [comment]
One minor comment, except that everything LGTM.

**Pull request merged**
